pages/glossary/non-printable-characters.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Non-printable characters
+key: non-printable-characters
+---
```

Comment:
Can you try and add the unambiguous/objective flags in all the new definitions (see #1085).